go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/jrick/bitset v1.0.0
 	github.com/jrick/logrotate v1.0.0
 	golang.org/x/crypto v0.0.0-20180718160520-a2144134853f
-	golang.org/x/sys v0.0.0-20180522224204-88eb85aaee56
+	golang.org/x/sys v0.0.0-20180816055513-1c9583448a9c
```

Comment:
This is more than just tidying, but shouldn't be an issue.

Reply:
Nevermind, this was the real minimum of the complete build anyways, due to other module depends.